containers/HomeHeaderRight/HomeHeaderRight.js

```diff
@@ -5,6 +5,7 @@ import React, { PureComponent } from 'react';
 import { View, Text, StyleSheet, TouchableOpacity } from 'react-native';
 import { connect } from 'react-redux';
 import { showAccountIssues } from 'actions';
+import I18n from 'utils';
```

Comment:
Looks like You should use `import { I18N }` and add import inside `utils/index.js` file